__tests__/endpoints.test.js

```diff
@@ -190,3 +191,44 @@ describe("PATCH /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  it("GET /api/reviews responds with an array and following properties, owner, title, review_id, category, review_img_url, created_at, votes, designer and comment_count", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(13);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  it("GET /api/reviews?cateogry=xyz responds with reviews filtered by cateogry xyz", () => {
+    return request(app)
+      .get("/api/reviews?category=euro game")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((review) => {
```

Comment:
remember to do length check before a forEach, so you're protected against false positives